dcos_installer/action_lib.py

```diff
@@ -360,8 +360,15 @@ def _add_prereqs_script(chain):
   exit 0
 fi
 
-if ([ x$dist != 'xrhel' ] && [ x$dist != 'xcentos' ]); then
-  echo "$dist is not supported. Only RHEL and CentOS are supported" >&2
+if ([ x$dist != 'xrhel' ] && [ x$dist != 'xcentos' ] && [ x$dist != 'xol' ]); then
+  echo "$dist is not supported. Only RHEL, CentOS and Oracle Enterprise Linux are supported" >&2
```

Comment:
Even if users are running Oracle Enterprise Linux with the Red Hat Compatible Kernel (RHCK) enabled, I'm not comfortable using the word supported here unless we actually have some basic automated test infrastructure to verify that it will work. @adam-mesos or @karlkfi is there some terminology that the PMC would approve for 'unsupported' DC/OS installation environments?